charlesinto/iReporter
# #162622211 Implemented Email Notifications

#### What does this PR do?

- Send Email when Admin update status of a record

#### Description of Task to be completed?
- Implement JWT authentication
- Verify User Role
- Update Status if Admin
- Set up nodemailer for nodejs
- Send email when status is updated

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `PATCH` request to `http://localhost:7000/api/v1/{type}/:id/status`
- You should receive a successful response

#### Any background context you want to provide?
The request should be a schema like:

`{status: "string"}`

`

#### What are the relevant pivotal tracker stories?
[#162622211](https://www.pivotaltracker.com/story/show/162622211)

#### Screenshots (if appropriate)


![image](https://user-images.githubusercontent.com/26615085/49917383-3b1fe000-fe9f-11e8-8096-622bee930c53.png)




#### Questions:
N/A
